tests/method-chain/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -6,10 +157,7 @@ exports[`test test.js 1`] = `
 ({}.a().b());
 ({}).a().b();
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-method()
-  .then(x => x)
-  [\"abc\"](x => x)
-  [abc](x => x);
+method().then(x => x)[\"abc\"](x => x)[abc](x => x);
```

Comment:
The point was, even if it fits, you want each call to be on a separate line. The code above is much harder to read than the expanded version before. If you're chaining function calls you want to read it vertically.